test/functional/api/vim_spec.lua

```diff
@@ -5507,4 +5507,156 @@ describe('API', function()
 
     n.assert_alive()
   end)
+
+  describe('nvim_cmd with plus flags', function()
+    it('handles +flags correctly', function()
```

Comment:
these it() cases can live in the existing describe() for nvim_cmd https://github.com/neovim/neovim/blob/91e116f3a6719e9dc7652f4e0a4b1760dfbf1af0/test/functional/api/vim_spec.lua#L4804